node/impl/full.go

```diff
@@ -117,4 +124,84 @@ func (n *FullNodeAPI) NodeStatus(ctx context.Context, inclChainStatus bool) (sta
 	return status, nil
 }
 
+// This method is exclusively used for Non Interactive Authentication based on SP Worker Address
+func (n *FullNodeAPI) FilIdSp(ctx context.Context, maddr address.Address) (string, error) {
+	fil_authhdr := "FIL-SPID-V0"
```

Comment:
Underscores are **also** frowned upon in golang 🤣 
https://go.dev/doc/effective_go#names